Modules/System/SharePoint/src/model/listitem/SharePointListItem.Table.al

```diff
@@ -0,0 +1,87 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+
+/// <summary>
+/// Holds information about list item entity.
+/// </summary>
+table 9103 "SharePoint List Item"
+{
+    Access = Public;
+    DataClassification = SystemMetadata;
+    Caption = 'SharePoint List Item';
+    TableType = Temporary;
+    Extensible = false;
+
+    fields
+    {
+        field(1; Guid; Guid)
```

Comment:
I'd call this "Id" instead of "Guid"

Reply:
Hi, as agreed we will keep the SharePoint naming.